### PR TITLE
Rename `random_seed` in `Enrollment` to `commitment`

### PR DESCRIPTION
**Pull request by kchulj**

### CODE REVIEW

**Comment by Geod24** [comment]
> Error: src/modules/storage/LedgerStorage.ts(363,32): error TS2339: Property 'commitment' does not exist on type 'Enrollment'.
> Error: src/modules/storage/LedgerStorage.ts(391,32): error TS2339: Property 'commitment' does not exist on type 'Enrollment'.

Almost there.

**Comment by kchulj** [comment]
> Almost there.

I am also updating `boa-sdk-ts`.

**Comment by TrustHenry** [comment]
version 0.0.36 of boa-SDK is now available.

File: src/modules/storage/LedgerStorage.ts [hunk 1]
```diff
@@ -1221,7 +1221,7 @@ export class LedgerStorage extends Storages
         `SELECT utxos.address,
                 enrollments.enrolled_at,
                 enrollments.utxo_key as stake,
-                enrollments.random_seed,
+                enrollments.commitment,
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts:1224]: https://github.com/bosagora/stoa/blob/71702931ae267107f3517ccdf0c96d79498bc151/src/Stoa.ts#L224
https://github.com/bosagora/stoa/blob/71702931ae267107f3517ccdf0c96d79498bc151/src/Stoa.ts#L318
This is also affected.

Comment (kchulj) [reply src/modules/storage/LedgerStorage.ts:1224]: Updated.

**Pull request merged**